pkg/gencmd/providers.go

```diff
@@ -9,13 +9,15 @@ import (
 )
 
 func ProvideGrapiCtx(ctx *Ctx) *grapicmd.Ctx         { return ctx.Ctx }
+func ProvideCtx(cmd *Command) *Ctx                   { return cmd.Ctx() }
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function ProvideCtx should have comment or be unexported